src/datasets/__init__.py

```diff
@@ -20,36 +20,26 @@
 __version__ = "1.18.5.dev0"
 
 import pyarrow
-from packaging import version as _version
-from pyarrow import total_allocated_bytes
+from packaging import version
 
 
-if _version.parse(pyarrow.__version__).major < 5:
+if version.parse(pyarrow.__version__).major < 5:
     raise ImportWarning(
         "To use `datasets`, the module `pyarrow>=5.0.0` is required, and the current version of `pyarrow` doesn't match this condition.\n"
         "If you are running this in a Google Colab, you should probably just restart the runtime to use the right version of `pyarrow`."
     )
 
+SCRIPTS_VERSION = "master" if version.parse(__version__).is_devrelease else __version__
+
+del pyarrow
+del version
+
 from .arrow_dataset import Dataset, concatenate_datasets
-from .arrow_reader import ArrowReader, ReadInstruction
-from .arrow_writer import ArrowWriter
```

Comment:
Here ArrowWriter and ArrowReader are removed from the top level module. I've seen that the lxmert example in `transformers` needs `datasets.ArrowWriter`, so I'm not sure if we should remove them

Reply:
Ok :)